template.tpl

```diff
@@ -400,6 +409,7 @@ const sendWidgetEvent = (eventName, eventsArr) => {
 
   for (const ev of eventsArr) {
     ev.queryId = storedEvent.queryId;
+    ev.gtm_v = SCRIPT_VERSION;
```

Comment:
@srfication `gtm_v` property is added to each event in logic here

Reply:
ah ic okok 👍